Resources/Reliability/qml/ReliabilityBayesian.qml

```diff
@@ -208,34 +208,40 @@ Form
 		Group
         {
 			CheckBox 
-			{ 
-				id:                 probTable
-				name:               "probTable"; 
-				label:              qsTr("Probability for: ")
-				childrenOnSameRow:  true
+            {
+                id:                 probTable
+                name:               "probTable";
+                label:              qsTr("Probability for:")
+                childrenOnSameRow:  true
```

Comment:
Please use tabs only in QML files. If you use Qt Creator for editing QML files, see https://doc.qt.io/qtcreator/creator-indenting-code.html#indenting-qml-files.

Reply:
So what do I do here? I thought I used tabs here.

Reply:
Well if you use RStudio to edit the files it will automatically change the tabs into spaces (unless you changed that in the settings). Otherwise, Qt Creator or some other text editor (Atom, VS code) should do the trick.

Reply:
Sorry, I am confused. You are saying I should open the qml file in Qt and use tabs? But where exactly? Just at the spot that you quoted here?

Reply:
In QML, we indent using tabs, never spaces. See also the [style guide](https://github.com/jasp-stats/jasp-desktop/blob/stable/Docs/development/jasp-qml-style-example.qml). So everywhere, not just the spot I quoted. Looking at your last commit I think you already figured that out though.